drivers/charger/sbs_charger/sbs_charger.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SBS_CHARGER_H_
+#define ZEPHYR_DRIVERS_SENSOR_SBS_CHARGER_H_
+
+#include <stdint.h>
+#include <zephyr/drivers/i2c.h>
+
+#define SBS_CHARGER_REG_SPEC_INFO     0x11
+#define SBS_CHARGER_REG_STATUS	      0x13
+#define SBS_CHARGER_REG_ALARM_WARNING 0x16
+
```

Comment:
Overall looks great!!  Is the plan to add SBS_CHARGER_REG_CHARGING_CURRENT, SBS_CHARGER_REG_CHARGING_VOLTAGE, and SBS_CHARGER_REG_MODE in a subsequent iteration, along with implementation of set_props?

Reply:
That would be correct. In addition to the 'set' properties for sbs_charger I have another simple charging IC (bq24190) in mind as a user to help flesh out the API beyond this.